test/plugins/test_ignore_long_lines.py

```diff
@@ -1,6 +1,7 @@
 """
 Test ignore-long-lines regex
 """
+# pylint: disable=toggle-no-name,toggle-non-boolean-default-value
```

Comment:
I'm confused why this is needed, and it makes me a little nervous about what will happen in edx-platform.  I will see if I can set up an edx-platform PR to see.

Reply:
The problem is that the test module contains purposefully wrong annotations, which are then linted by pylint and flagged as errors. This should not happen outside of test fixtures.